### PR TITLE
Adding KENG data plane performance python tests to otg-examples

### PR DESCRIPTION
**Pull request by dosarudaniel**
Adding performance tests samples in snappi/pytest-examples. These are performance high rate unidirectional, bidirectional and rfc2544 test scripts. The proposed files configures and deploy our Ixia-C DPDK based Traffic Generator.

Workflow:
1. Run setup.sh on both test agents:
`sudo ./setup.sh ens64 ens66`
2. Run tests:
`./unidirectional.sh` / `./bidirectional.sh` / `./rfc_2544.sh`

### CODE REVIEW

File: snappi/pytest-examples/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+## Document version  
+Make sure you have the latest version of this readme file by executing:
+```
+git pull  # in the otg-examples repository, origin: https://github.com/open-traffic-generator/otg-examples.git
+```
+## Description
+The goal of this repository is to help you setup, deploy and run test using Keysight Elastic Network Generator.
```
Comment (isgmano) [inline snappi/pytest-examples/README.md:7]: This is a too broad statement. There are other examples of how to use KENG as well. What is the specific usecase this example solves? Is this examples with pytest?

File: snappi/pytest-examples/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+cd ~/keng-tests/deployment
+./setup.sh
+docker-compose up -d
+```
+
+To validate that the deployment was succesful, run on VM1 :
+```
+root@keng-agent:/home/ixia/ixia-c-tests/deployment# docker ps
+CONTAINER ID   IMAGE                                                              COMMAND                  CREATED         STATUS         PORTS     NAMES
+5ac4e8831d8e   ghcr.io/open-traffic-generator/licensed/ixia-c-controller:latest   "./bin/controller --…"   4 seconds ago   Up 3 seconds             Ixia-c-Controller
```
Comment (isgmano) [inline snappi/pytest-examples/README.md:74]: what is the licensed controller?

**Review by isgmano — CHANGES_REQUESTED** [review]
Please resolve the comments. 
Mention what is unique in these examples and deployment.

File: snappi/pytest-examples/py/test_unidirectional.py
[the remark this answers is not in the document]
Comment (dosarudaniel) [reply snappi/pytest-examples/py/test_unidirectional.py:1]: These are performance high rate unidirectional, bidirectional and rfc2544 test scripts. The proposed files configures and deploy our Ixia-C DPDK based Traffic Generator.

**Review by isgmano — APPROVED** [review]

**Review by bortok — CHANGES_REQUESTED** [review]
@dosarudaniel could you please confirm that the plan is to use these scripts by @armdupre in his cloud-based dpdk lab example? asking because although you have diagrams, the readme is empty. so people would not be able to understand this contribution without an example that is using it.

also, could you please rename the subfolder from `pytest-examples` into `rfc2544` or `data-plane-performance`? I think your scripts are much more usable as is vs being a simple example

**Review by armdupre — APPROVED** [review]

File: snappi/data-plane-performance/deployment/docker-compose.yaml
Comment (bortok) [inline snappi/data-plane-performance/deployment/docker-compose.yaml:1]: Please remove this file, has all sorts of artifacts internal to Keysight

File: snappi/data-plane-performance/deployment/setup.sh
Comment (bortok) [inline snappi/data-plane-performance/deployment/setup.sh:1]: Do we need this, or will this be replaced by terraform lab setup?

File: snappi/data-plane-performance/results/throughput_results_rfc2544_4_flows_upstream.json
Comment (bortok) [inline snappi/data-plane-performance/results/throughput_results_rfc2544_4_flows_upstream.json:1]: do we need the results file?

File: snappi/data-plane-performance/settings.json
Comment (bortok) [inline snappi/data-plane-performance/settings.json:1]: please remove private IPs from the configuration file

**Review by bortok — CHANGES_REQUESTED** [review]
I made a few more notes, please address them before merging

File: snappi/data-plane-performance/deployment/setup.sh
Comment (dosarudaniel) [reply snappi/data-plane-performance/deployment/setup.sh:1]: We need this, it is not related to terraform

File: snappi/data-plane-performance/deployment/docker-compose.yaml
Comment (dosarudaniel) [reply snappi/data-plane-performance/deployment/docker-compose.yaml:1]: I removed the internal artifacts

File: snappi/data-plane-performance/settings.json
Comment (dosarudaniel) [reply snappi/data-plane-performance/settings.json:1]: Done.

File: snappi/data-plane-performance/results/throughput_results_rfc2544_4_flows_upstream.json
Comment (dosarudaniel) [reply snappi/data-plane-performance/results/throughput_results_rfc2544_4_flows_upstream.json:1]: Removed

File: snappi/data-plane-performance/deployment/setup.sh [hunk 3]
```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Run this script with elevated privileges
+
+TRAFFIC_ENGINE_VERSION="1.6.0.167"
+CONTROLLER_VERSION="1.3.0-2"
+LICENSE_SERVER_VERSION="latest"
```
Comment (bortok) [inline snappi/data-plane-performance/deployment/setup.sh:7]: No longer used?

Comment (dosarudaniel) [reply snappi/data-plane-performance/deployment/setup.sh]: Indeed, fixed

**Review by bortok — APPROVED** [review]

**Pull request merged**